packages/auth/test/helpers/integration/helpers.ts

```diff
@@ -76,9 +80,15 @@ export function getTestInstance(requireEmulator = false): Auth {
   return auth;
 }
 
-export async function cleanUpTestInstance(auth: Auth): Promise<void> {
+export async function cleanUpTestInstance(
+  auth: Auth,
+  tests?: string
+): Promise<void> {
   await auth.signOut();
-  await (auth as IntegrationTestAuth).cleanUp();
+
+  if (typeof tests === 'undefined') {
+    await (auth as IntegrationTestAuth).cleanUp();
```

Comment:
If we are doing with skipping cleanup of hard-coded email, I prefer modifying https://github.com/firebase/firebase-js-sdk/blob/2a12a01c5113a3b0476d1775f4bd84697a5eb339/packages/auth/test/helpers/integration/helpers.ts#L70 to skip deleting users that have the string "donotdelete" in them. We use this same string to construct our hardcoded email.

Reply:
+1, this feels clearer